cmd/kubeadm/app/apis/kubeadm/v1alpha2/types.go

```diff
@@ -277,6 +277,14 @@ type JoinConfiguration struct {
 	// the security of kubeadm since other nodes can impersonate the master.
 	DiscoveryTokenUnsafeSkipCAVerification bool `json:"discoveryTokenUnsafeSkipCAVerification"`
 
+	// Controlplane flag specifies that the joining node should host an additional
+	// controlplane instance.
+	Controlplane bool `json:"controlplane,omitempty"`
```

Comment:
Although `controlplane` looks better to me, perhaps, for the sake of consistency with the rest of the fields here, we should rename these:

`Controlplane` -> `ControlPlane`
`controlplane` -> `controlPlane`